test/testdrive/materialization-lag.td

```diff
@@ -11,6 +11,11 @@
 #
 # These tests rely on testdrive's retry feature, as they query introspection
 # relations whose data might not be immediately available.
+#
+# Note that when running under cloudtest, it's quite common to see lags of over
+# 1 second even when all replicas are healthy. So we need to build some
+# tolerances into these checks that verify whether induced lag is present. A
+# threshold value of 2 seconds seems to work fine.
```

Comment:
@nrainer-materialize Is this something you have seen before?

Reply:
No, I haven't.